dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerTest.java

```diff
@@ -1145,7 +1145,7 @@ public void testFindLinkageProblems_referenceToJava11Method() throws IOException
             new ClassFile(jars.get(0), "com.google.protobuf.TextFormat"),
             null,
             methodSymbol,
-            "java.nio.CharBuffer");
+            "java.nio.Buffer");
```

Comment:
Because of the missing equals method, this test was not using correct expected value.